xbmc/input/InputManager.cpp

```diff
@@ -491,7 +492,12 @@ bool CInputManager::OnEvent(XBMC_Event& newEvent)
         || (actionId >= ACTION_MOUSE_START && actionId <= ACTION_MOUSE_END))
         CApplicationMessenger::Get().SendAction(CAction(actionId, 0, newEvent.touch.x, newEvent.touch.y, newEvent.touch.x2, newEvent.touch.y2), WINDOW_INVALID, false);
     else
-      CApplicationMessenger::Get().SendAction(CAction(actionId), WINDOW_INVALID, false);
+    {
+      if (actionId == ACTION_BUILT_IN_FUNCTION && actionString.length() > 0)
```

Comment:
int actionID, float amount1 /* = 1.0f */ <- thats the c'tor which only has an id - the default for amount1 makes volumeup/down work (and maybe others too).